algo_training/train_v2_two_tower.py

```diff
@@ -0,0 +1,164 @@
+import os
```

Comment:
(#) J'ai l'impression qu'il y a beaucoup de duplication avec les autres fichiers de train, il y a peut être une amélioration à tirer. qu'est-ce que tu en penses ?

Reply:
Je pense qu'on itère pas mal est qu'à la fin il en restera qu'un donc pas forcément besoin de refacto train
Par contre un ticket pour suppr les anciens modèles a été créé déjà